quizzes/midterm.md

```diff
@@ -1,62 +1,87 @@
-1. Circle all 5 of the problems with the code below.
-
-	```ruby
-	class Monkey < Monkey
-	  def init(my_vars)
-	    my_vars = my_vars
-	  end
-
-	  private
-	  def eat_banana
-	    "Om Om Om, oooh aaaah aaaah"
+#Mid-Term : Ruby
+##Gary McGinnis
+
+1: Circle all 5 of the problems with the code below.
```

Comment:
6/10 points. You answered 3 correctly.

Answers
1. Cannot inherit from yourself
2. `def init` should be `def initialize`
3. Should not set local variable to self
4. Cannot access private method
5. No arguments when instantiating new object